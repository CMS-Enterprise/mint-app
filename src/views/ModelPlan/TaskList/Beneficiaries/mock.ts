import GetModelPlanBeneficiaries from 'queries/GetModelPlanBeneficiaries';
import { GetModelPlanBeneficiaries_modelPlan_beneficiaries as ModelPlanBeneficiariesType } from 'queries/types/GetModelPlanBeneficiaries';
import {
  BeneficiariesType,
  ConfidenceType,
  FrequencyType,
  OverlapType,
  SelectionMethodType,
  TaskStatus,
  TriStateAnswer
} from 'types/graphql-global-types';

export const beneficiaryMockData: ModelPlanBeneficiariesType = {
  __typename: 'PlanBeneficiaries',
  id: '123',
  modelPlanID: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905',
  beneficiaries: ['OTHER' as BeneficiariesType],
  beneficiariesOther: 'Other Note',
  beneficiariesNote: '',
  treatDualElligibleDifferent: 'YES' as TriStateAnswer,
  treatDualElligibleDifferentHow: 'Differ text',
  treatDualElligibleDifferentNote: '',
  excludeCertainCharacteristics: 'YES' as TriStateAnswer,
  excludeCertainCharacteristicsCriteria: '',
  excludeCertainCharacteristicsNote: '',
  numberPeopleImpacted: 100,
  estimateConfidence: 'COMPLETELY' as ConfidenceType,
  confidenceNote: '',
  beneficiarySelectionNote: '',
  beneficiarySelectionOther: '',
  beneficiarySelectionMethod: ['HISTORICAL' as SelectionMethodType],
  beneficiarySelectionFrequency: 'ANNUALLY' as FrequencyType,
  beneficiarySelectionFrequencyNote: '',
  beneficiarySelectionFrequencyOther: '',
  beneficiaryOverlap: 'YES_NO_ISSUES' as OverlapType,
  beneficiaryOverlapNote: '',
  precedenceRules: '',
  createdBy: '',
  createdDts: '',
  modifiedBy: '',
  modifiedDts: '',
  status: TaskStatus.COMPLETE
};

const beneficiaryMock = [
  {
    request: {
      query: GetModelPlanBeneficiaries,
      variables: { id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905' }
    },
    result: {
      data: {
        modelPlan: {
          id: '1029',
          modelName: 'My excellent plan that I just initiated',
          generalCharacteristics: beneficiaryMockData
        }
      }
    }
  }
];

export default beneficiaryMock;
