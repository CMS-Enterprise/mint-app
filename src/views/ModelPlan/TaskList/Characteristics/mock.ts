import GetModelPlanCharacteristics from 'queries/GetModelPlanCharacteristics';
import { GetModelPlanCharacteristics_modelPlan_generalCharacteristics as ModelPlanCharacteristicsType } from 'queries/types/GetModelPlanCharacteristics';
import {
  GeographyType,
  KeyCharacteristic,
  TaskStatus
} from 'types/graphql-global-types';

export const charactersticMockData: ModelPlanCharacteristicsType = {
  __typename: 'PlanGeneralCharacteristics',
  id: '123',
  modelPlanID: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905',
  isNewModel: false,
  existingModel: 'Second Plan',
  resemblesExistingModel: false,
  resemblesExistingModelWhich: [],
  resemblesExistingModelHow: '',
  resemblesExistingModelNote: '',
  hasComponentsOrTracks: true,
  hasComponentsOrTracksDiffer: 'Differ text',
  hasComponentsOrTracksNote: 'Component note',
  alternativePaymentModel: false,
  alternativePaymentModelTypes: [],
  alternativePaymentModelNote: '',
  keyCharacteristics: ['OTHER' as KeyCharacteristic],
  keyCharacteristicsOther: 'Key other note',
  keyCharacteristicsNote: '',
  collectPlanBids: false,
  collectPlanBidsNote: '',
  managePartCDEnrollment: false,
  managePartCDEnrollmentNote: '',
  planContactUpdated: false,
  planContactUpdatedNote: '',
  careCoordinationInvolved: false,
  careCoordinationInvolvedDescription: '',
  careCoordinationInvolvedNote: '',
  additionalServicesInvolved: true,
  additionalServicesInvolvedDescription: 'Lots of additional services',
  additionalServicesInvolvedNote: '',
  communityPartnersInvolved: false,
  communityPartnersInvolvedDescription: '',
  communityPartnersInvolvedNote: '',
  geographiesTargeted: true,
  geographiesTargetedTypes: ['OTHER' as GeographyType],
  geographiesTargetedTypesOther: 'Other geography type',
  geographiesTargetedAppliedTo: [],
  geographiesTargetedAppliedToOther: '',
  geographiesTargetedNote: '',
  participationOptions: false,
  participationOptionsNote: '',
  agreementTypes: [],
  agreementTypesOther: '',
  multiplePatricipationAgreementsNeeded: false,
  multiplePatricipationAgreementsNeededNote: '',
  rulemakingRequired: true,
  rulemakingRequiredDescription: 'Yes rulemaking is required',
  rulemakingRequiredNote: '',
  authorityAllowances: [],
  authorityAllowancesOther: '',
  authorityAllowancesNote: '',
  waiversRequired: false,
  waiversRequiredTypes: [],
  waiversRequiredNote: '',
  createdBy: '',
  createdDts: '',
  modifiedBy: '',
  modifiedDts: '',
  status: TaskStatus.COMPLETE
};

const charactersticMock = [
  {
    request: {
      query: GetModelPlanCharacteristics,
      variables: { id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905' }
    },
    result: {
      data: {
        modelPlan: {
          modelName: 'My excellent plan that I just initiated',
          generalCharacteristics: charactersticMockData
        }
      }
    }
  }
];

export default charactersticMock;
