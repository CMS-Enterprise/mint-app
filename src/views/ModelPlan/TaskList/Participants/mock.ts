import GetModelPlanParticipantsAndProviders from 'queries/GetModelPlanParticipantsAndProviders';
import { GetModelPlanProvidersAndParticipants_modelPlan_participantsAndProviders as ModelPlanParticipantsAndProvidersType } from 'queries/types/GetModelPlanProvidersAndParticipants';
import {
  ParticipantRiskType,
  ParticipantsIDType,
  ParticipantsType,
  ProviderAddType,
  TaskStatus
} from 'types/graphql-global-types';

export const participantsAndProvidersData: ModelPlanParticipantsAndProvidersType = {
  __typename: 'PlanParticipantsAndProviders',
  id: '123',
  modelPlanID: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905',
  participants: [
    'MEDICARE_PROVIDERS' as ParticipantsType,
    'STATES' as ParticipantsType,
    'OTHER' as ParticipantsType
  ],
  medicareProviderType: 'Oncology Providers',
  statesEngagement:
    'States will determine administration specific to the state',
  participantsOther: 'The candy people',
  participantsNote: '',
  participantsCurrentlyInModels: null,
  participantsCurrentlyInModelsNote: '',
  modelApplicationLevel: 'c92.00 and c92.01',
  expectedNumberOfParticipants: 350,
  estimateConfidence: null,
  confidenceNote: '',
  recruitmentMethod: null,
  recruitmentOther: '',
  recruitmentNote: '',
  selectionMethod: [],
  selectionOther: '',
  selectionNote: '',
  communicationMethod: [],
  communicationNote: '',
  participantAssumeRisk: true,
  riskType: 'OTHER' as ParticipantRiskType,
  riskOther: 'Programmatic Risk',
  riskNote: '',
  willRiskChange: null,
  willRiskChangeNote: '',
  coordinateWork: null,
  coordinateWorkNote: '',
  gainsharePayments: null,
  gainsharePaymentsTrack: null,
  gainsharePaymentsNote: '',
  participantsIds: ['OTHER' as ParticipantsIDType],
  participantsIdsOther: 'Candy Kingdom Operations Number',
  participantsIDSNote: '',
  providerAdditionFrequency: null,
  providerAdditionFrequencyOther: '',
  providerAdditionFrequencyNote: '',
  providerAddMethod: ['OTHER' as ProviderAddType],
  providerAddMethodOther: 'Competitive ball-room dancing, free for all',
  providerAddMethodNote: '',
  providerLeaveMethod: [],
  providerLeaveMethodOther: '',
  providerLeaveMethodNote: '',
  providerOverlap: null,
  providerOverlapHierarchy: '',
  providerOverlapNote: '',
  createdBy: '',
  createdDts: '',
  modifiedBy: '',
  modifiedDts: '',
  status: TaskStatus.COMPLETE
};

const participantsAndProvidersMock = [
  {
    request: {
      query: GetModelPlanParticipantsAndProviders,
      variables: { id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905' }
    },
    result: {
      data: {
        modelPlan: {
          modelName: 'My excellent plan that I just initiated',
          participantsAndProviders: participantsAndProvidersData
        }
      }
    }
  }
];

export default participantsAndProvidersMock;
