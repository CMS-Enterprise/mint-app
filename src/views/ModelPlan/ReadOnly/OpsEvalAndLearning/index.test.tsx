import React from 'react';
import { MemoryRouter, Route } from 'react-router-dom';
import { MockedProvider } from '@apollo/client/testing';
import { render, screen, waitFor } from '@testing-library/react';
import { DataToSendParticipantsType } from 'gql/gen/graphql';
import i18next from 'i18next';
import Sinon from 'sinon';

import { modelID, opsEvalAndLearningMocks as mocks } from 'data/mock/readonly';

import ReadOnlyOpsEvalAndLearning from './index';

describe('Read Only Model Plan Summary -- Operations, Evaluation, and Learning', () => {
  // Stubing Math.random that occurs in Truss Tooltip component for deterministic output
  Sinon.stub(Math, 'random').returns(0.5);

  it('renders without errors', async () => {
    render(
      <MemoryRouter
        initialEntries={[
          `/models/${modelID}/read-only/operations-evaluation-and-learning`
        ]}
      >
        <MockedProvider mocks={mocks} addTypename={false}>
          <Route path="/models/:modelID/read-only/operations-evaluation-and-learning">
            <ReadOnlyOpsEvalAndLearning modelID={modelID} />
          </Route>
        </MockedProvider>
      </MemoryRouter>
    );

    await waitFor(() => {
      expect(screen.getByTestId('tasklist-tag')).toHaveTextContent(
        'In progress'
      );
      expect(
        screen.getByText(
          i18next.t<string>(
            `opsEvalAndLearning:dataToSendParticicipants.options.${DataToSendParticipantsType.BENEFICIARY_LEVEL_DATA}`
          )
        )
      ).toBeInTheDocument();
    });
  });
  it('matches snapshot', async () => {
    const { asFragment } = render(
      <MemoryRouter
        initialEntries={[
          `/models/${modelID}/read-only/operations-evaluation-and-learning`
        ]}
      >
        <MockedProvider mocks={mocks} addTypename={false}>
          <Route path="/models/:modelID/read-only/operations-evaluation-and-learning">
            <ReadOnlyOpsEvalAndLearning modelID={modelID} />
          </Route>
        </MockedProvider>
      </MemoryRouter>
    );
    await waitFor(() => {
      expect(screen.getByTestId('tasklist-tag')).toHaveTextContent(
        'In progress'
      );
      expect(
        screen.getByText(
          i18next.t<string>(
            `opsEvalAndLearning:dataToSendParticicipants.options.${DataToSendParticipantsType.BENEFICIARY_LEVEL_DATA}`
          )
        )
      ).toBeInTheDocument();
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
