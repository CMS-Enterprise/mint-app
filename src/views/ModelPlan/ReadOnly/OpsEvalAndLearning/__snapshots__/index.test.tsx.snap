// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Read Only Model Plan Summary -- Operations, Evaluation, and Learning matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="read-only-model-plan--ops-eval-and-learning"
    data-testid="read-only-model-plan--ops-eval-and-learning"
  >
    <div
      class="display-flex flex-justify flex-align-start"
    >
      <h2
        class="margin-top-0 margin-bottom-4"
      >
        Operations, evaluation, and learning
      </h2>
      <span
        class="model-plan-task-list__task-tag line-height-body-1 text-bold mint-no-print bg-warning"
        data-testid="tasklist-tag"
      >
        In progress
      </span>
    </div>
    <div
      class="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light"
    >
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--will-another-agency-or-state-help-design-operate-the-model margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Will another Agency or State help design/operate the model?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              Yes, we will partner with states
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Other
            </li>
            <ul
              data-testid="other-entry"
            >
              <li
                class="font-sans-md line-height-sans-4"
              >
                <em
                  class="text-base"
                >
                  Other not specified
                </em>
              </li>
            </ul>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--what-stakeholders-do-you-plan-to-communicate-with margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            What stakeholders do you plan to communicate with?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              Beneficiaries
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Participants
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Providers
            </li>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--do-you-plan-to-use-a-helpdesk margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Do you plan to use a helpdesk?
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--what-contractors-will-support-your-model margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                What contractors will support your model?
              </p>
              <ul
                class="margin-y-0 padding-left-3"
              >
                <li
                  class="font-sans-md line-height-sans-4"
                >
                  May have separate contractors for different implementation functions
                </li>
                <li
                  class="font-sans-md line-height-sans-4"
                >
                  Other
                </li>
                <ul
                  data-testid="other-entry"
                >
                  <li
                    class="font-sans-md line-height-sans-4"
                  >
                    <em
                      class="text-base"
                    >
                      Other not specified
                    </em>
                  </li>
                </ul>
              </ul>
            </div>
          </div>
        </div>
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--in-what-capacity-will-they-support-your-model margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                In what capacity will they support your model?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light"
    >
      <h3>
        IDDOC Operations
      </h3>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--are-you-planning-to-use-iddoc-support margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Are you planning to use IDDOC support?
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
    </div>
    <div
      class="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light"
    >
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--will-you-establish-a-benchmark-to-capture-performance margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Will you establish a benchmark to capture performance?
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--will-you-compute-performance-scores margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Will you compute performance scores?
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--will-you-make-risk-adjustments-to-performance-scores margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                Will you make risk adjustments to performance scores?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--will-you-make-risk-adjustments-to-feedback-results margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                Will you make risk adjustments to feedback results?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--will-you-make-risk-adjustments-to-payments margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                Will you make risk adjustments to payments?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--will-you-make-risk-adjustments-to-others margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                Will you make risk adjustments to others?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--will-participants-be-able-to-appeal-performance-scores margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                Will participants be able to appeal performance scores?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--will-participants-be-able-to-appeal-feedback-results margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                Will participants be able to appeal feedback results?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--will-participants-be-able-to-appeal-payments margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                Will participants be able to appeal payments?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
        <div
          class="desktop:grid-col-6"
          data-testid="grid"
        >
          <div
            class="desktop:grid-col-12"
            data-testid="grid"
          >
            <div
              class="read-only-section read-only-section--will-participants-be-able-to-appeal-others margin-bottom-3"
            >
              <p
                class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
              >
                Will participants be able to appeal others?
              </p>
              <p
                class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
              >
                <em
                  class="text-base"
                >
                  No answer entered
                </em>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light"
    >
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--what-type-of-evaluation-approach-are-you-considering margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            What type of evaluation approach are you considering?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              Interrupted time series
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Other
            </li>
            <ul
              data-testid="other-entry"
            >
              <li
                class="font-sans-md line-height-sans-4"
              >
                <em
                  class="text-base"
                >
                  Other not specified
                </em>
              </li>
            </ul>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--is-chronic-conditions-warehouse--ccw--involved-in-the-model margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Is Chronic Conditions Warehouse (CCW) involved in the model?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              Yes, for evaluation
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Other
            </li>
            <ul
              data-testid="other-entry"
            >
              <li
                class="font-sans-md line-height-sans-4"
              >
                <em
                  class="text-base"
                >
                  Other not specified
                </em>
              </li>
            </ul>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--what-data-do-you-need-to-monitor-the-model margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            What data do you need to monitor the model?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              Clinical data
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Medicare claims
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Other
            </li>
            <ul
              data-testid="other-entry"
            >
              <li
                class="font-sans-md line-height-sans-4"
              >
                <em
                  class="text-base"
                >
                  Other not specified
                </em>
              </li>
            </ul>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--what-data-will-you-send-to-participants margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            What data will you send to participants?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              Baseline/historical data
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Beneficiary-level data
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Other
            </li>
            <ul
              data-testid="other-entry"
            >
              <li
                class="font-sans-md line-height-sans-4"
              >
                <em
                  class="text-base"
                >
                  Other not specified
                </em>
              </li>
            </ul>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--does-the-model-require-that-identifiable-claim-and-claim-line-feed--cclfs--data-need-to-be-shared-with-participants margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Does the model require that identifiable Claim and Claim Line Feed (CCLFs) data need to be shared with participants?
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
    </div>
    <div
      class="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light"
    >
      <h3>
        Chronic Conditions Warehouse (CCW)
      </h3>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--will-you-need-to-send-files-between-the-ccw-and-other-applications margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Will you need to send files between the CCW and other applications?
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="desktop:grid-col-12"
          data-testid="grid"
        >
          <div
            class="read-only-section read-only-section--do-you-know-which-applications-will-be-on-the-other-sides-of-the-file-transfers margin-bottom-3"
          >
            <p
              class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
            >
              Do you know which applications will be on the other sides of the file transfers?
            </p>
            <p
              class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
            >
              <em
                class="text-base"
              >
                No answer entered
              </em>
            </p>
          </div>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--will-you-use-the-ccw-to-distribute-files-to-and-from-model-participants margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Will you use the CCW to distribute files to and from model participants?
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
    </div>
    <div
      class="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light"
    >
      <h3>
        Data Sharing, Collection, and Reporting Timing and Frequency
      </h3>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--data-sharing-starts margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Data sharing starts
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--how-often-do-you-anticipate-sharing-data margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            How often do you anticipate sharing data?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              Daily
            </li>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--data-collection-starts margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Data collection starts
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--how-often-do-you-anticipate-collecting-data margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            How often do you anticipate collecting data?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              Monthly
            </li>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--quality-reporting-starts margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Quality reporting starts
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
    </div>
    <div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--will-the-model-have-a-learning-strategy margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            Will the model have a learning strategy?
          </p>
          <ul
            class="margin-y-0 padding-left-3"
          >
            <li
              class="font-sans-md line-height-sans-4"
            >
              We plan to use an IT platform (Connect)
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              No, we will not have a learning strategy
            </li>
            <li
              class="font-sans-md line-height-sans-4"
            >
              Other
            </li>
            <ul
              data-testid="other-entry"
            >
              <li
                class="font-sans-md line-height-sans-4"
              >
                <em
                  class="text-base"
                >
                  Other not specified
                </em>
              </li>
            </ul>
          </ul>
        </div>
      </div>
      <div
        class="desktop:grid-col-12"
        data-testid="grid"
      >
        <div
          class="read-only-section read-only-section--what-challenges-do-you-anticipate-during-model-design-and-implementation margin-bottom-3"
        >
          <p
            class="text-bold margin-y-0 font-body-sm line-height-sans-4 text-pre-line"
          >
            What challenges do you anticipate during Model design and implementation?
          </p>
          <p
            class="margin-y-0 font-body-md line-height-sans-4 text-pre-line"
          >
            <em
              class="text-base"
            >
              No answer entered
            </em>
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
